### DISCUSSION TITLE
MorphoSource is Back! Temporary Instability May Occur

### DISCUSSION THREAD

**Discussion by JulieWinchester** [MorphoSource/Community#114] [Announcements]
Hello MorphoSource users! We are back from maintenance, and the repository is now running on an entirely new infrastructure. You may encounter slight instability over the next few days, as we tweak the system for the new infrastructure. Thank you for your patience.

**Comment by cgoatley** [comment]
Hi there,
Hope all the updates go well.
Just FYI, my downloads are all finishing/crashing at around 1 Gb when trying to download larger files.

**Comment by chelsiesnipes** [comment]
Double FYI - mine are also failing at 1GB, and the file is invalid when I try to unzip it.

**Comment by allisonbronson** [comment]
We're having the same issue, downloads failing around 1GB and/or the files that download are empty folders.

**Comment by JulieWinchester** [comment]
Thanks for the bug reports! We're investigating.

**Comment by JulieWinchester** [comment]
@cgoatley @chelsiesnipes @allisonbronson We seem to have fixed the issue where downloads have been interrupting at 1GB. Thanks again for your reports! 

We are possibly still experiencing a related issue where download resume is not working. So if a download is interrupted due to external bandwidth interruptions due to internet outages or otherwise, that download can't currently be resumed. We're working on this issue as well. Thanks for your patience.
